jit/src/main/java/com/justintime/jit/entity/FoodRestaurant.java

```diff
@@ -0,0 +1,21 @@
+package com.justintime.jit.entity;
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "food_restaurant")
+public class FoodRestaurant {
```

Comment:
Shall we make the FoodRestaurant class as DTO instead of entity?

Reply:
If this is menu item, we do need it as each restaurant would have distinct menu item, so it can be entity

Reply:
A separate table has been made for menu item , we dont need the entity here